jenkins/job-configs/kubernetes-jenkins/kubernetes-e2e-kops-aws.yaml

```diff
@@ -0,0 +1,67 @@
+- job-template:
+    name: 'kubernetes-e2e-kops-{suffix}'
+    node: '{jenkins_node}'
+    description: '{description} Test owner: {test-owner}.'
+    jenkins_node: 'e2e'
+    disabled: '{obj:disable_job}'
+    properties:
+        - build-discarder:
+            days-to-keep: 7
+    triggers:
+        - reverse:
+            jobs: '{trigger-job}'
```

Comment:
kubernetes-e2e-gce.yaml has multiple values for trigger-job:

`kubernetes-e2e-gce` uses [`kubernetes-build`](https://github.com/kubernetes/test-infra/blob/master/jenkins/job-configs/kubernetes-jenkins/kubernetes-e2e-gce.yaml#L50) while kubernetes-e2e-gce-federation uses [`kubernetes-federation-build`](https://github.com/kubernetes/test-infra/blob/master/jenkins/job-configs/kubernetes-jenkins/kubernetes-e2e-gce.yaml#L401)

Your job has only one value for trigger-job right now, therefore the intermediate variable is unnecessary.

If you write `jobs: 'kubernetes-build'` the next person has the information right in front of their eyes instead of needing to go hunt down where trigger-job is defined.